setup.sh

```diff
@@ -8,19 +8,15 @@ echo 'Installing macOS packages...'
 brew update
 brew install node
 brew install mongodb
-brew install grunt
```

Comment:
We can't remove `brew install grunt` since the dev script still calls `grunt watch`.